code/modules/food_and_drink/alcohol.dm

```diff
@@ -265,8 +265,8 @@
 	alt_filled_state = 1
 	heal_amt = 1
 	g_amt = 60
-	initial_volume = 60
-	initial_reagents = list("wine"=40,"charcoal"=20)
+	initial_volume = 120
+	initial_reagents = list("wine"=80,"charcoal"=40)
```

Comment:
keep to 100u

Reply:
Like this? Reverted to the original amount of chemicals inside with the initial volume as 100.